### PR TITLE
Print Friendly Styles

### LINKED ISSUE #373: Print before final submission
As an eApp Applicant I want to be able to print my data **after** my form data has been confirmed and **before** it has been submitted so that I can keep a printed copy for my personal record or provide to my employer.

Acceptance criteria:
- [x] printing will follow decided-on web print output

Brought over from #294

### PR DESCRIPTION
**Pull request by floresj**
Resolves #373 

Added temporary Print menu item just so we can test out. Eventually, we can remove it and add functionality to the final submission page.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -104,6 +106,17 @@ function convert () {
...
 
+function printCss () {
+  'use strict'
+  return gulp
+    .src('./src/sass/print.scss')
+    .pipe(sass({
+      includePaths: [ paths.sass.vars ]
+    }))
+    .pipe(concat(paths.printCss))
+    .pipe(gulp.dest(paths.destination.css))
```
Comment (bmallred) [inline gulpfile.js]: So is the `eqip.print.css` file getting all of the other styling as well? Is that necessary or can we just leverage browser layering?

Comment (floresj) [reply gulpfile.js]: ahh I forgot to update this. Going to remove.

**Comment by bmallred** [comment]
Should the navigation section reflect if there are errors?

image

**Comment by bmallred** [comment]
@Jay-Neighbours-I72 should we make the **print** form look like [this](https://github.com/truetandem/e-QIP-prototype/issues/1523#issuecomment-316494801) right now?

e-app-comp-submit-7

**Comment by bmallred** [comment]
Merging this to `master` with the expectation that @floresj (or someone) will need it to work on #1699

**Pull request merged**